analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -84,6 +91,7 @@
     public static final String APPLY_GME_REPOS = "buildscript { apply from: new File(buildscript.getSourceFile().getParentFile(),\"gme-repos.gradle\"), to: buildscript }";
     public static final String GME_PLUGINCONFIGS = "gme-pluginconfigs.gradle";
     public static final String NAME = "generateAlignmentMetadata";
+    public static final String MANIPULATION_PME_BASENAME = "manipulation-pme";
```

Comment:
Where did you get this name from - I didn't think this was agreed in the ticket?

Reply:
I wasn't sure if it was going to be `alignmentReport.json` or involve renaming `gme.json`. Also, what should the text version of the report be called? I think ` `alignmentReport.txt` makes sense in that case.